Resources/Prototypes/SimpleStation14/Damage/modifier_sets.yml

```diff
@@ -31,3 +31,8 @@
     Shock: 1.25
     Radiation: 1.3
 
+- type: damageModifierSet
+  id: SlimeReagents
+  coefficients:
+    Poison: -1
+  minimumDamageToDamage: -999
```

Comment:
Add a negative flat reduction. The effects of this would be to increase healing before reversing it, making it more deadly, and to lower damage before reversing it, making it less healing. I think this is good for balance.

Reply:
["flat reductions cannot heal you"](https://github.com/Simple-Station/Parkstation/blob/slimin/Content.Shared/Damage/DamageSpecifier.cs#L122)

Reply:
Well, you're right, but also it seems to be your problem now :P

Either way, I actually meant a standard flat reduction- but it should *also* increase the power of healing.